sdk/eventhub/azure-eventhubs/HISTORY.md

```diff
@@ -1,5 +1,28 @@
 # Release History
 
+## 2019-11-04 5.0.0b5
+
+**Breaking changes**
+
+- `EventHubClient` has been split into two separate clients: `EventHubProducerClient` and `EventHubConsumerClient`.
```

Comment:
Please mention all the module which are not plublic anymore (see release note of azure-core 1.0.0b4 for example)

Reply:
We have only removed `EventHubClient`, `EventHubProducerClient`, `EventHubConsumerClient`.

Reply:
right, I've updated the changelog.